spec/std/mime_spec.cr

```diff
@@ -15,7 +15,16 @@ module MIME
 end
 
 describe MIME do
+  before_each do
+    MIME.reset!
+  end
+
+  after_each do
+    MIME.reset!
+  end
```

Comment:
Why not use `around_each` instead?

Reply:
Does it make a difference? `around_each` is for when you need to share state between setup and teardown steps. There's no locally shared state here, so can just use before and after hooks.